GeosCore/hco_interface_gc_mod.F90

```diff
@@ -5016,6 +5029,22 @@ SUBROUTINE Compute_Sflx_for_Vdiff( Input_Opt,  State_Chm, State_Diag,      &
              ENDIF
           ENDIF
 
+          ! Dry deposition velocity (cm/s):
+          IF ( State_Diag%Archive_DryDepVel ) THEN
+             S = State_Diag%Map_DryDepVel%id2slot(ND)
+             IF ( S > 0 ) THEN
+                State_Diag%DryDepVel(:,:,S) = dvel(:,:,N)
+             ENDIF
+          ENDIF
+
+          ! Satellite diagnostic dry deposition velocity (cm/s):
+          IF ( State_Diag%Archive_SatDiagnDryDepVel ) THEN
+             S = State_Diag%Map_SatDiagnDryDepVel%id2slot(ND)
+             IF ( S > 0 ) THEN
+                State_Diag%SatDiagnDryDepVel(:,:,S) = dvel(:,:,N)
+             ENDIF
+          ENDIF
+
```

Comment:
As I noted above, putting these here would only work for non-local mixing. Full mixing does not call this routine.

Reply:
This should now be fixed in commit d9ee2e2.